### DISCUSSION TITLE
A way to include an internal URI when working with internal Evergreen Libraries

### DISCUSSION THREAD

**Discussion by Nucklez** [EUCPilots/evergreen-module#455] [Ideas]
Would it be possible to include an internal URL instead of defaulting to the local or UNC path that was specified when you run the Invoke-EvergreenLibraryUpdate?   I've been updating an internal web server, and would love to have the internal URL somewhere in the Evergreen.  I can manipulate the Path variable in the json files to point to the internal URL now, but was curious if there was a way or idea to add an internal URL/replace the URI with the internal URL or modify the path in the JSON files with an internal URL?

**Comment by aaronparker** [comment]
I'm currently working on away to publish to blob storage on an Azure storage account. I'd use azcopy in this case, and then have the library use a https URI in the path.

What's the internal server running?

**Reply by Nucklez** [reply]
It's running IIS.  I'm just dropping files in a folder structure and using Chocolatey to grab them from the URL instead of a file share.
